### PR TITLE
Fix multiple version in annotation condition handling

### PR DESCRIPTION
**Pull request by boyez-joggin**
By a _in []_ syntax instead of the current
For #1436

### CODE REVIEW

**Comment by dmaicher** [comment]
This is also fixed in https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1438

File: Routing/Loader/Reader/RestActionReader.php [hunk 1]
```diff
@@ -352,16 +352,16 @@ private function getCondition(\ReflectionMethod $method, RouteAnnotation $annota
         $condition = $annotation->getCondition();
 
         if (!empty($this->versions)) {
-            $versionCondition = "request.attributes.get('version') == (";
+            $versionCondition = "request.attributes.get('version') in [";
             $first = true;
             foreach ($this->versions as $version) {
```
Comment (dmaicher) [inline Routing/Loader/Reader/RestActionReader.php]: I don't see how this loop could generate something like you mentioned in https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1436

`((<version> or <version>) or <version>)`

Comment (boyez-joggin) [inline Routing/Loader/Reader/RestActionReader.php]: Sorry there were one set of parenthesis which came from symfony language auto generation.
The real result is this:

```
request.attributes.get('version') == (<version> or <version> or <version>)
```

which then auto-generates to:

```
request.attributes.get('version') == ((<version> || <version>) || <version>)
```
like I added to the related issue #1436 

So with my fix we get:
```
request.attributes.get('version') in [<version>, <version>, <version>]
```
which auto-generates to:
```
in_array(request.attributes.get('version'), array(0 => <version>, 1 => <version>, 2 => <version>))
```

Sorry for the mix, my bad :x

Comment (dmaicher) [inline Routing/Loader/Reader/RestActionReader.php]: Ok now I undertand :+1:  Thx for the explanation. The old solution with the `or` is totally wrong indeed. I also fixed that problem in my other PR though:

https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1438

**Comment by Ener-Getick** [comment]
Closed in favor of https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1438

**Pull request closed without merging**